ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-ospf.j2

```diff
@@ -194,8 +194,8 @@
 
 ### Router OSPF Areas
 
-| Process ID | Area | Area Type | Filter Networks | Filter Prefix List | Additional Config Options |
-| ---------- | ---- | --------- | --------------- | ------------------ | ------------------------- |
+| Process ID | Area | Area Type | Filter Networks | Filter Prefix List | Additional Options |
+| ---------- | ---- | --------- | --------------- | ------------------ | ------------------ |
 {%         for process in router_ospf.process_ids | arista.avd.natural_sort %}
```

Comment:
The follow the coding standard `for process` should be `for process_id`, I believe this was missed in the past, but let's correct this now to be consistent!